plugins/gradle/src/index.ts

```diff
@@ -100,8 +97,12 @@ export default class GradleReleasePluginPlugin implements IPlugin {
   }
 
   /** update gradle version and commit */
-  private updateGradleVersion = async (version: string, commitMsg?: string) => {
-    if (this.properties.buildDuringRelease) {
+  private updateGradleVersion = async (
+    version: string,
+    commitMsg?: string,
+    buildFlag: boolean = true
```

Comment:
cool, yup, makes since. any objection to changing the name of this method ... seems like it's not just updating the version now... plus, it would also allow gradle-release (pre|post) hooks to run if they exist too.  If not now, we can also do it later.   I don't have a better name ATM anyways.